helper.linux.fish

```diff
@@ -353,7 +354,9 @@ function downloadStarter
 end
 
 function downloadSyncer
-  runInContainer -e DOWNLOAD_SYNC_USER=$DOWNLOAD_SYNC_USER $UBUNTUBUILDIMAGE $SCRIPTSDIR/downloadSyncer.fish $THIRDPARTY_BIN $argv
+  rm -f $WORKDIR/work/ArangoDB/build/install/usr/sbin/arangosync $WORKDIR/work/ArangoDB/build/install/usr/bin/arangosync
+  runInContainer -e DOWNLOAD_SYNC_USER=$DOWNLOAD_SYNC_USER $UBUNTUBUILDIMAGE $SCRIPTSDIR/downloadSyncer.fish $THIRDPARTY_SBIN $argv
+  ln -s ../sbin/arangosync $WORKDIR/work/ArangoDB/build/install/usr/bin/arangosync
```

Comment:
I would prefer to use full paths, otherwise the function has to be called within a certain context. Knowing the current directory was one of the biggest problems while working on the documentation script.

Reply:
I think an absolute path might break a destdir install. Leaving it as it is